keyboards/rainkeebs/plop/plop.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 Regan Palmer (@rainkeebs)
```

Comment:
This file should be removed. (Assuming the matrix data in `info.json` is correct)